src/cloudinary-placeholder.component.ts

```diff
@@ -40,16 +41,25 @@ export class CloudinaryPlaceHolder implements AfterContentChecked {
   }
 
   getPlaceholderImage() {
-    const placeholderImageOptions = {
-      'vectorize': {effect: 'vectorize', quality: 1},
-      'pixelate': {effect: 'pixelate', quality: 1, fetch_format: 'auto'},
-      'blur': {effect: 'blur:2000', quality: 1, fetch_format: 'auto'},
-      'solid': [
+    if (this.type === 'predominant' && this.itemHeight && this.itemWidth) {
+      this.predominantTransform = [
+        {width: 'iw_div_2', aspect_ratio: 1, crop: 'pad', background: 'auto'},
+        {crop: 'crop', width: 10, height: 10, gravity: 'north_east'},
+        {width: '1', height: '1', crop: 'fill'},
+        {fetch_format: 'auto', quality: 'auto'}]
+    } else {
+      this.predominantTransform  = [
         {width: 'iw_div_2', aspect_ratio: 1, crop: 'pad', background: 'auto'},
         {crop: 'crop', width: 10, height: 10, gravity: 'north_east'},
         {width: 'iw', height: 'ih', crop: 'fill'},
         {fetch_format: 'auto', quality: 'auto'}]
     }
+    const placeholderImageOptions = {
+      'vectorize': {effect: 'vectorize', quality: 1},
+      'pixelate': {effect: 'pixelate', quality: 1, fetch_format: 'auto'},
+      'blur': {effect: 'blur:2000', quality: 1, fetch_format: 'auto'},
+      'predominant': this.predominantTransform
+    }
```

Comment:
@strausr , we said we're going to move that out of here... 
Please set as a member (instead of the _predominantTranform_) and in the above "if" condition, set the relevant `this.placeholderImageOptions.predominant`